statistics-production/ud.qmd

```diff
@@ -319,6 +319,40 @@ As with file names, you should avoid any special characters; for example, the fo
 
 Variable names should ideally be kept below 25-35 characters as long names are often cut off when viewing the data file and generally fail to get the information required across to users. It is a balance between giving enough information so it's clear what it refers to and giving so much that it's unhelpful. Remember to make use of your public data guidance and methodology for expanding on details.
 
+Titles should use abbreviations only when necessary to reduce the length of the title if required.
+
+#### Indicator names
+
+Most indicators should be reducible to a simple context / title (e.g. schools, pupils, students, teachers, starts, apprenticeships, expenditure, income, etc) and a data type (e.g. count, sum, percent, score, average, median, fte, etc). Assuming this ideal (tidy data structure) case, the preferred layout is:
+
+{title}_{data type}
+
+For example:
+
+starts_count, starts_sum, starts_percent, starts_average, starts_median, absence_percent, pupils_count, pupils_percent
+
+Whilst data producers should generally aim to fit the basic layout above, there may be circumstances in which additional flags need to be included that can’t be placed in Filters. In these circumstances, the guidance is to follow the below ordering:
+
+{title}_{levels}_{above / below}_{exclusivity}_{additional}_{data type}
```

Comment:
Think this line would benefit from an example, suspect a few stats producers might get lost / misunderstand it without one

Reply:
Fair. I didn't bother putting an example as I didn't want to implicitly sanction people using this structure where they shouldn't. Will see if I can come up with an example where it feels vaguely legitimate o use this and not go against tidy data principles.

Reply:
That's a pretty reasonable approach actually - if you can't find a good example without spending too much time then we can leave it without

Reply:
Ok, have gone with that, but added an extra line saying get in touch if you're planning on this route, so we can either dissuade or gather legitimate examples!